Common/Resources/Schemes/Script/Devanagari.map

```diff
@@ -94,10 +96,12 @@ VOWEL	VOCALIC L	090C
 VOWEL	SHORT E	090E
 VOWEL	E	090F
 VOWEL	AI	0910
+VOWEL	CANDRA O	0911
 VOWEL	SHORT O	0912
 VOWEL	O	0913
 VOWEL	AU	0914
 VOWEL	VOCALIC RR	0960
 VOWEL	VOCALIC LL	0961
+VOWEL	CANDRA A	0972
```

Comment:
Shouldn't this be called CANDRA E?

Reply:
It's a bit weird, but CANDRA E is ऍ, and CANDRA A is ॲ (Marathi uses the latter), so my choice of CANDRA A was deliberate.
Also, it seems to me that the CANDRA variants are right above their respective SHORT variants (unless you meant something else).
Let me know if anything else needs to be changed!